README.md

```diff
@@ -2,197 +2,230 @@
 
 [![Build Status](https://travis-ci.org/vtex/toolbelt.svg?branch=master)](https://travis-ci.org/vtex/toolbelt) [![npm version](https://img.shields.io/npm/v/vtex.svg?style=flat)](https://www.npmjs.com/package/vtex) [![Greenkeeper badge](https://badges.greenkeeper.io/vtex/toolbelt.svg)](https://greenkeeper.io/)
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX IO platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
```

Comment:
I chose a shorter version to incorporate the "shortcuts concept". The real section titles are larger: created to communicate a little bit about what is inside, as a small Tweet — they are not just a way to start a text section. 
The shortcuts should be... shorter. 

Any specific reason for using identical text at the "Summary"?

Reply:
Yeah, it was just because I was expecting something when I clicked and was redirected to something ""else"" and for a fraction of seconds I was a little confused. But I understand what you want.

Reply:
For now, I'll just use the shorter but I'll ask other designers to read and tell me what they think! Maybe more people feel the same.